FabricExample/patches/react-native+0.71.0-rc.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/react-native/ReactAndroid/cmake-utils/ReactNative-application.cmake b/node_modules/react-native/ReactAndroid/cmake-utils/ReactNative-application.cmake
```

Comment:
plz remove this file

Reply:
Done in https://github.com/software-mansion/react-native-gesture-handler/pull/2313/commits/f6df763638b1b41cd2aaa1cf08d2b396e55676e3.